src/test/java/com/google/sps/SchedulerTest.java

```diff
@@ -54,6 +55,33 @@ public void setUp() {
     return Arrays.asList(mon, wed, fri);
   }
 
+  /**
+   * Returns a list of times on monday and friday, to be used in section constructors.
+   */
+  public List<TimeRange> monFri(int hour, int minute, int durationMinutes) {
```

Comment:
These day/time helper functions are a bit repetitive.

Instead of defining separate functions for each combination, you could create variables:

```
private static final int[] MON_WED = {TimeRange.MONDAY, TimeRange.WEDNESDAY};
...
```
And then pass those days into a single function:

```
private List<TimeRange> createSectionTimes(int[] days, int hour, int minute, int durationMinutes) {
  List<TimeRange> timeRanges = new ArrayList<>();
  for (int day : days) {
    TimeRange timeRange = TimeRange.fromStartDuration(day, hour, minute, durationMinutes);
    timeRanges.add(timeRange);
  }
  return timeRanges;
}
```

Reply:
I agree, this definitely could use some cleaning up. However, given where we're at w/ development & time restrictions, it might be better to focus dev time on finishing features/fixing bugs instead of cleaning up tests (especially since these particular tests will effectively not change from here on out). What are your thoughts?

Reply:
Sure, just wanted to point out that the five helper functions could be replaced with a single one. I understand the time constraints however so I'll approve this as is.